### PR TITLE
Create Million_KMs-Cycled-Per-Year.rmd

### PR DESCRIPTION
**Pull request by natesheehan**
- RE issue #59 
- Rough first draft discussing how to calculate, compare and use the PCT models

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Thanks for this @natesheehan. I think there's an issue with vignette titles:

https://github.com/ITSLeeds/pct/runs/1578485229#step:9:151

File: vignettes/Million_KMs-Cycled-Per-Year.rmd [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+title: "Estimating million km's cycled per zone in England and Wales regions"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing cycling potential estimates in UK cities}
```
Comment (Robinlovelace) [inline vignettes/Million_KMs-Cycled-Per-Year.rmd:5]: This should be updated to: Estimating million km's cycled per zone in England and Wales regions

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Looks good, needs a different title for the tests to pass. Also could you rename the file, e.g. from

```
Million_KMs-Cycled-Per-Year.rmd 
```

to

```
km-cycled.Rmd
```

Short and sweet!

**Review by natesheehan — COMMENTED** [review]
Ah yep sounds good

**Comment by Robinlovelace** [comment]
Many thanks @natesheehan, looking good. It's still complaining about duplicate title, not 100% sure what it's going on about tbh or how to fix it... 2 options: I merge now and fix up on my branch or you give it another bash. You call!

**Comment by natesheehan** [comment]
Hm let me try deleting this PR and creating a new one // if not have a go on your branch.

**Pull request closed without merging**